api/api.go

```diff
@@ -112,6 +112,11 @@ func (a *API) getAs(o interface{}, tpl string, p *params) error {
 	if err := xml.Unmarshal(buf, o); err != nil {
 		return uof.Notice("unmarshal", err)
 	}
+	// assert FixtureRsp and inject raw
+	fx, ok := o.(*fixtureRsp)
```

Comment:
Ovome type assertion nije mjesto u getAs. getAs funkcionira za razlicite tipove i neovisna je o tipu podataka s kojim radi. Ovime to rusis. Cinis ju specificnom za jedan slucaj. 
Za sva druge slucajeve ovaj assertion ce pucati. To znas unaprijed i nema potrebe da forsiras u assert playera, marketa u fixture.

Reply:
Hvala na odgovoru!
Treba mi raw zasad fixture koji dolaze sa `/sports/{language}/sport_events/{urn_type}:{id}/fixture.xml`
Ali, mogao bih staviti i raw za markets i players.

Pucati kao panickati?